workshops/running-amazon-ec2-workloads-at-scale/user-data.txt

```diff
@@ -14,6 +14,7 @@ runcmd:
   - [ sh, -c, "./home/ec2-user/install auto" ]
   - [ sh, -c, "systemctl enable codedeploy-agent.service" ]
   - [ sh, -c, "systemctl start codedeploy-agent.service" ]
+  - [ sh, -c, "yum -y install stress-ng" ]
```

Comment:
This bit should be removed from the user-data.txt.  stress-ng is installed as part of the code-deploy stage. This can be removed. Codedeploy does eventually come to: https://github.com/awslabs/ec2-spot-workshops/blob/master/workshops/running-amazon-ec2-workloads-at-scale/codedeploy/scripts/install_dependencies.sh.

That codedeploy script and approach approach has the benefit of using strict execution (set -e) to capture errors and mark as unhealthy entries with errors, then recover them as required if the deployment goes wrong.

Reply:
Hi Carlos, I modified the commit to change only the file that contains the doc.
For the record I was not able to reproduce the problem with stress-ng.
Thanks!